reference/docs-conceptual/community/contributing/powershell-style-guide.md

```diff
@@ -0,0 +1,492 @@
+  uses the **Object** parameter to ...
+  ```
+
+  For more information, see [Hyperlinks](#hyperlinks) section of this article.
+
+### Images
+
+The syntax to include an image is:
+
+```markdown
+![[alt text]](<folderPath>)
+
+Example:
+![Introduction](./media/overview/Introduction.png)
+```
+
+Where `alt text` is a brief description of the image and `<folder path>` is a relative path to the
+image. Alternate text is required for screen readers for the visually impaired. It is also useful in
+case there is a site bug where the image cannot be rendered.
+
+Images should be stored in a `media/<article-name>` folder within the folder containing the article.
+Images should not be shared between articles. Create a folder that matches the filename of your
+article under the `media` folder. Copy the images for that article to that new folder. If an image
+is used by multiple articles, each image folder must have a copy of that image file. This practice
+prevents a change to an image in one article affecting another article.
+
+The following image file types are supported: *.png", *.gif", *.jpeg", *.jpg", *.svg
+
+### Markdown extensions supported by Open Publishing
+
+The [Microsoft Docs Authoring Pack](/contribute/how-to-write-docs-auth-pack) contains tools that
+support features unique to our publishing system. Alerts are a Markdown extension to create block
+quotes that render on docs.microsoft.com with colors and icons that indicate the significance of the
+content. The following alert types are supported:
+
+```markdown
+> [!NOTE]
+> Information the user should notice even if skimming.
+
+> [!TIP]
+> Optional information to help a user be more successful.
+
+> [!IMPORTANT]
+> Essential information required for user success.
+
+> [!CAUTION]
+> Negative potential consequences of an action.
+
+> [!WARNING]
+> Dangerous certain consequences of an action.
+```
+
+These alerts look like this on docs.microsoft.com:
+
+> [!NOTE]
+> Information the user should notice even if skimming.
+
+> [!TIP]
+> Optional information to help a user be more successful.
+
+> [!IMPORTANT]
+> Essential information required for user success.
+
+> [!CAUTION]
+> Negative potential consequences of an action.
+
+> [!WARNING]
+> Dangerous certain consequences of an action.
+
+## Hyperlinks
+
+- Avoid using bare URLs. Links should use Markdown syntax `[friendlyname](url-or-path)`
+- Bare URLs may be used when necessary, but should be enclosed in backticks. For example:
+
+  ```markdown
+  By default, if you do not specify this parameter, the DMTF standard resource URI
+  `http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/` is used and the class name is appended to it.
+  ```
+
+- Links must have a friendly name, usually the title of the linked topic
+- All items in the "related links" section at the bottom should be hyperlinked.
+- Do not use backticks, bold, or other markup inside the brackets of a hyperlink.
+
+### Linking to other content
```

Comment:
It was mentioned elsewhere, but do we want to add here that they should also be HTTPS?

Reply:
It was a few like below but I added it here.